dockerfiles/ink-ci-linux/Dockerfile

```diff
@@ -21,23 +21,28 @@ RUN	set -eux; \
 # The supported Rust nightly version must support the following components
 # to allow for a functioning CI pipeline:
 #
-#  - cargo
-#  - rustfmt
-#  - clippy
-#  - miri (in the future)
-# see on https://rust-lang.github.io/rustup-components-history/
-	rustup default nightly-2020-05-07; \
-# install wasm32 target for this toolchain
+#  - cargo: General build tool.
+#  - rustfmt: Rust formatting tool.
+#  - clippy: Rust linter.
+#  - miri: Rust interpreter with additional safety checks.
+#
+# We also need to install the wasm32-unknown-unknown target to test
+# ink! smart contracts compilation for this target architecture.
+#
+# Only Rust nightly builds supporting all of the above mentioned components
+# and targets can be used for this docker image.
+#
+# Visit https://rust-lang.github.io/rustup-components-history/ to see which
+# components are available for which nightly build.
```

Comment:
maybe we should move some comments to the readme?

Reply:
i think the comments are better here since they are for the developers that are touching the file directly, not for the user of the dockerfile. it is neat to have information about how to update next to what you want to update in a file.